views/jira-configuration.hbs

```diff
@@ -106,226 +130,107 @@
             <div class="jiraConfiguration__syncRetryModalOverlay"></div>
           </div>
 
-          {{#if successfulConnections}}
-          <div class="jiraConfiguration__tableContainer">
-            <table class="jiraConfiguration__table aui">
-              <thead class="jiraConfiguration__table__head">
-                <tr class="jiraConfiguration__table__head_row">
-                  <th class="jiraConfiguration__table__head__title">Connected organization</th>
-                  <th class="jiraConfiguration__table__head__title">Repository access</th>
-                  <th class="jiraConfiguration__table__head__title">Backfill status
-                    <span class="jiraConfiguration__table__infoIcon aui-icon aui-iconfont-info-filled" id="sync-status-modal-btn">
-                      Information
-                    </span>
-                  </th>
-                  <th class="jiraConfiguration__table__head__title">Settings</th>
-                </tr>
-              </thead>
+          <div class="jiraConfiguration__optionContainer">
```

Comment:
We only want to show this if a user has both cloud and server connected. It doesn't make much sense to me to have the option to click 'All' or 'Github Cloud'/'GitHub Enterprise Server' if I'm only connected to one version type.

- [ ] hide the lozenges unless a user is connected to both cloud and server

Reply:
Logic has been corrected. This `optionContainer` will be shown only if there are both GH cloud and GHE server apps. Otherwise it will remain hidden.